CSH/test_plots/plot_msid_latest_conda.py

```diff
@@ -20,57 +26,68 @@
+	'AIOCV':{'msid_ls':['AIOAP5CV','AIOBP5CV'], 'units': ['V', 'V'], 'weight':1, 'title':'IOE Converter Voltage'},
+	'ARWBT':{'msid_ls':['ARWA1BT', 'ARWA2BT', 'ARWA3BT', 'ARWA4BT', 'ARWA5BT', 'ARWA6BT'], 'units': ['F', 'F', 'F', 'F', 'F', 'F'], 'weight':1, 'title':'RWA Bearing Temperature'},
+	'AWDCV':{'msid_ls':['AWD1CV5V', 'AWD2CV5V', 'AWD3CV5V', 'AWD4CV5V', 'AWD5CV5V', 'AWD6CV5V'], 'units': ['V', 'V', 'V', 'V', 'V', 'V'], 'weight':1, 'title':'WDE Converter Voltage'},
+	'AWDTQ':{'msid_ls':['AWD1TQI', 'AWD2TQI', 'AWD3TQI', 'AWD4TQI','AWD5TQI', 'AWD6TQI'], 'units': ['A', 'A', 'A', 'A', 'A', 'A'], 'weight':1, 'title':'Wheel Torque Current'},
+	'ASPEV':{'msid_ls':['ASPEB5CV','ASPEA5CV'], 'units': ['V', 'V'], 'weight':1, 'title': 'SPE Converter Voltage'},
+	'AVDCV':{'msid_ls':['AVD1CV5V', 'AVD2CV5V'], 'units': ['V', 'V'], 'weight':1, 'title': 'VDE Converter Voltage'},
+	'1CB_T':{'msid_ls':['1CBAT', '1CBBT'], 'units': ['C', 'C'], 'weight':1, 'title': 'Camera Body Temp'},
+	'1CR_T':{'msid_ls':['1CRAT', '1CRBT'], 'units': ['C', 'C'], 'weight':1, 'title': 'Cold Radiator Temp'},
+	'1D_MZT':{'msid_ls':['1DEAMZT', '1DPAMZT'], 'units': ['C', 'C'], 'weight':1, 'title': '-Z Panel Temp'},
+	'1WR_T':{'msid_ls':['1WRAT', '1WRBT'], 'units': ['C', 'C'], 'weight':1, 'title': 'Warm Radiator Temp'},
+	'Hrc_temps':{'msid_ls':['2CHTRPZT', '2DCENTRT', '2DTSTATT', '2FRADPYT'], 'units': ['C', 'C', 'C', 'C'], 'weight':1, 'title': 'HRC Temps'},
+	'Pline_1_8':{'msid_ls':['PLINE01T','PLINE02T','PLINE03T','PLINE04T','PLINE05T','PLINE06T','PLINE07T','PLINE08T'], 'units': ['F', 'F', 'F', 'F', 'F', 'F', 'F', 'F'], 'weight':1, 'title': 'PLINE 01-08 Temps'},
+	'Pline_9_16':{'msid_ls':['PLINE09T','PLINE10T','PLINE11T','PLINE12T','PLINE13T','PLINE14T','PLINE15T','PLINE16T'], 'units': ['F', 'F', 'F', 'F', 'F', 'F', 'F', 'F'], 'weight':1, 'title': 'PLINE 01-08 Temps'},
+	'Mups_tank':{'msid_ls':['PMTANK1T', 'PMTANK2T', 'PMTANK3T'], 'units': ['F', 'F', 'F'], 'weight':1, 'title': 'Mups Tank Temp'},
+	'Fuel_tank':{'msid_ls':['PFTANK1T', 'PFTANK2T'], 'units': ['F', 'F'], 'weight':1, 'title': 'Fuel Tank Temp'}
 
 	}
+"""
+MSID_GROUP_SELECTION = ['Pcaditv', 'Aosymom', 'Aorate', 'Aodithr', 'Hrcelec', 'Spcelec', 'Pcadgdrift', 'Gratgen', 'Ycurrent', \
+'Sc_temp', 'Mups', 'Spcelecb', 'Spceleca', 'Spcelec_pwr', 'Pcadgrate', 'Sys_temps', 'Zmodel_temps', 'Mups_load', 'Tank_temp', \
+'Quat', 'Blkhd', 'Aosares', 'Psmc', 'Dpa', 'EB1', 'EB2', 'EB3', 'Scs_acis', 'TB1', 'TB2', 'TB3', 'AACHT', 'ACPV', 'ADEV', \
+'AGWSV', 'IRUBT', 'Pcaditt', 'AIOCV', 'ARWBT', 'AWDCV', 'AWDTQ', 'ASPEV', 'AVDCV', '1CB_T', '1CR_T', '1D_MZT', '1WR_T', \
+'Hrc_temps', 'Pline_1_8', 'Pline_9_16', 'Mups_tank', 'Fuel_tank'] #Selections of which MSID groupings to plot. Default to all.
+"""
+MSID_GROUP_SELECTION = list(PLOT_LOOKS.keys())#Selections of which MSID groupings to plot. Default to all.
+
+CONSIDER_TIME = False #whether or not to use the plot_time.json file to determine whcih MSID_groups to generate the plot for.
```

Comment:
Before this PR to update the python script, there was a .plot_these.txt file which stored cutoff dates in which, if the script was run at a time past this date, it would no longer create the corresponding msid groupings script and div plotting files. This is one of the reasons none of the state of health plots were being generated anymore, as they were being deliberately cutoff from this file. 

I kept this functionality around while checking to see if this plot_time check was still desirable but I haven't found any reason to turn off our ability to plot the current MSIDs in the SOH pages. Removing this check and the plot_time.json file.